libs/elements/layout/texts/src/lib/elewa-text-section/elewa-text-section.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input } from '@angular/core';
+import { HeroText } from '@elewa-website/models/schema/ui/texts';
+
+@Component({
+  selector: 'elewa-website-elewa-text-section',
+  templateUrl: './elewa-text-section.component.html',
+  styleUrls: ['./elewa-text-section.component.scss'],
+})
+
+// text section displays a series of texts
+
+export class ElewaTextSectionComponent {
+  
+  // texts to load
+  @Input() heroTexts : HeroText[] = []
```

Comment:
The issue is here. Should just be of type HeroText. Also don't initialise it use the bang operator. Check the codebase on how it's been done in other modules